src/main/java/org/folio/service/export/LocalFileSystemExportService.java

```diff
@@ -101,19 +106,42 @@ public void exportSrsRecord(Pair<List<String>, Integer> marcToExport, ExportPayl
     }
   }
 
+  private String[] getDataAfterClosingQuote(String jsonRecord) {
+    // First, remove all data inside double quotes.
```

Comment:
From me point it is not needed to comment sim much, as it is clear form the code.